ANGULAR/Task_2/User-Management-App/src/app/shared/user-list/user-list.component.html

```diff
@@ -0,0 +1,4 @@
+<div *ngFor="let user of userList">
+    <app-user [user] = "user" [route]="route" *ngIf="(route === 'active' && !user.isDeleted)||(route === 'deleted' && user.isDeleted)||(route == 'manage')"></app-user>
```

Comment:
This is a case where we should take code formatting into the picture.
Move the *ngIf condition to a variable in the component. 

Any structural conditions should not be this long